src/main.ts

```diff
@@ -318,7 +318,7 @@ export function prettifyExecutionGraphResult(executionGraphResult: Object): void
     if (task["tests"]) {
```

Comment:
IMO we should apply the filter here.

Reply:
I made a change here, what do you think?